node/tests/contract_test.rs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) 2019, MASQ (https://masq.ai) and/or its affiliates. All rights reserved.
+
+use ethabi;
+use futures::Future;
+use masq_lib::blockchains::chains::Chain;
+use web3;
+use web3::contract::Options;
+
+fn assert_contract(
+    blockchain_url: &str,
+    chain: &Chain,
+    expected_token_name: &str,
+    expected_decimals: u32,
+) {
+    let (_event_loop, transport) = web3::transports::Http::new(blockchain_url).unwrap();
+    let web3 = web3::Web3::new(transport);
+    let address = chain.rec().contract;
+    let min_abi_json = r#"[{
+        "constant": true,
+        "inputs": [],
+        "name": "name",
+        "outputs": [
+            {
+                "name": "",
+                "type": "string"
+            }
+        ],
+        "payable": false,
+        "stateMutability": "view",
+        "type": "function"
+    },
+        {
+        "constant": true,
+        "inputs": [],
+        "name": "decimals",
+        "outputs": [
+            {
+                "name": "",
+                "type": "uint8"
+            }
+        ],
+        "payable": false,
+        "stateMutability": "view",
+        "type": "function"
+    }]"#;
+    let abi = ethabi::Contract::load(min_abi_json.as_bytes()).unwrap();
+    let contract = web3::contract::Contract::new(web3.eth(), address, abi);
```

Comment:
This isn't critical, but I'd like to see a more descriptive name here. It's not really the contract, it's the pair of results from the two function calls. I understand that those two results are intimately connected with the contract, but they're not the same thing. Reading the code might be easier with a better name.